usr.bin/vtfontcvt/vtfontcvt.8

```diff
@@ -22,12 +24,12 @@
 .\" OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 .\" SUCH DAMAGE.
 .\"
-.Dd July 19, 2022
+.Dd June 4, 2024
 .Dt VTFONTCVT 8
 .Os
 .Sh NAME
 .Nm vtfontcvt
-.Nd "convert font files for use by the video console"
+.Nd convert font files for use by the system video console
```

Comment:
In an old man/working/vtfontcvt.8 I have `convert font files for use by the virtual terminal console` and I think that's better for new users search results, but I think `system video console` is better for specifically and consistently describing it.

Reply:
vtfontcvt is specific to vt so this is one place where it might make sense to mention -- but perhaps as part of the `.Sh DESCRIPTION` below rather than part of `.Nd`

Reply:
Latest push reverts proposed changes to this file in favor of a future PR working on this page specifically.